### DISCUSSION TITLE
Custom domain for github pages site only works for home page

### DISCUSSION THREAD

**Discussion by walkermoskop** [community/community#23217] [Other Feature Feedback, Questions, & Ideas]
<p>I’m pointing a domain originally set up through squarespace to my github pages site, and it works as expected for the homepage, <a href="http://walkermoskop.com" rel="noopener nofollow ugc">walkermoskop.com</a>, but all other pages lead to a 404 error. When running dig <a href="http://walkermoskop.com/about" rel="noopener nofollow ugc">walkermoskop.com/about</a> from the command line, I can see that unlike the home page, it’s not using the github pages IP addresses (I assume it’s using squarespace or an affiliated IP). Any ideas on how to address this? Or is this more of a squarespace issue? FWIW, within the squarespace advanced DNS settings, I added the following entries.</p>
<p><img src="https://user-images.githubusercontent.com/7673606/181075630-29c3b399-7360-47e1-af38-7a21b5cab82b.png" alt="image" data-base62-sha1="xvvFfJCPjGZbQ2RCZe7mdgVOXuB" width="553" height="284"></p>
<p>(Apologies for not linking the repo, which can be found through my profile, but as a new user I can’t post more than two links.)</p>
<p>Thanks in advance for any suggestions.</p>

**Comment by AlfredoRamos** [comment, marked as answer]
<p>You need to remove the <code>#</code> from the <code>pages</code> in the <code>include</code>, in your <code>_config.yml</code> file</p>
<aside class="onebox githubblob">
  <header class="source">
      <a href="https://github.com/walkermoskop/walkermoskop.github.io/blob/e70ff6cecba2194bafae209c3299cab660ddfd90/_config.yml#L35" target="_blank" rel="noopener nofollow ugc">github.com</a>
  </header>
  <article class="onebox-body">
    <h4><a href="https://github.com/walkermoskop/walkermoskop.github.io/blob/e70ff6cecba2194bafae209c3299cab660ddfd90/_config.yml#L35" target="_blank" rel="noopener nofollow ugc">walkermoskop/walkermoskop.github.io/blob/e70ff6cecba2194bafae209c3299cab660ddfd90/_config.yml#L35</a></h4>
<pre class="onebox"><code class="lang-yml"><ol class="start lines" start="25" style="counter-reset: li-counter 24 ;">
<li># Build settings</li>
<li>markdown: kramdown</li>
<li>remote_theme: mmistakes/minimal-mistakes</li>
<li># Outputting</li>
<li>permalink: /:categories/:title/</li>
<li># paginate: 5 # amount of posts to show</li>
<li># paginate_path: /page:num/</li>
<li>timezone: # https://en.wikipedia.org/wiki/List_of_tz_database_time_zones</li>
<li>
<li>include:</li>
<li class="selected">  # - _pages</li>
<li>
<li># Exclude from processing.</li>
<li># The following items will not be processed, by default. Create a custom list</li>
<li># to override the default setting.</li>
<li># exclude:</li>
<li>#   - Gemfile</li>
<li>#   - Gemfile.lock</li>
<li>#   - node_modules</li>
<li>#   - vendor/bundle/</li>
<li>#   - vendor/cache/</li>
</ol></code></pre>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>Should be:</p>
<pre><code class="lang-yaml">include:
  - _pages
</code></pre>
<p>More information specific for your theme:</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://mmistakes.github.io/minimal-mistakes/docs/pages/" target="_blank" rel="noopener nofollow ugc">Minimal Mistakes</a>
  </header>
  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/4165935/181075635-5f605c8b-c34c-493c-9759-68eca9780151.png" class="thumbnail onebox-avatar" width="500" height="500">

<h3><a href="https://mmistakes.github.io/minimal-mistakes/docs/pages/" target="_blank" rel="noopener nofollow ugc">Working with Pages</a></h3>

<p>Suggestions and Front Matter defaults for working with pages.</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

<p>You could also follow the standard way of creating pages by moving those pages in your root directory, however I’m not sure if some changes would be required for the theme you are using.</p>
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <img src="https://jekyllrb.com/favicon.ico" class="site-icon" width="32" height="32">
      <a href="https://jekyllrb.com/docs/pages/" target="_blank" rel="noopener nofollow ugc" title="07:17AM - 07 January 2021">Jekyll • Simple, blog-aware, static sites – 7 Jan 21</a>
  </header>
  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/345;"><img src="https://user-images.githubusercontent.com/4165935/181075636-9009d4a5-85e3-44da-9881-7868cd78919b.png" class="thumbnail" width="690" height="345" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_690x345.png, https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_1035x517.png 1.5x, https://aws1.discourse-cdn.com/github/original/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/9/b/9bb05134cb18fa7763a4c7cf4a020c5b9c7ce870_2_10x10.png"></div>

<h3><a href="https://jekyllrb.com/docs/pages/" target="_blank" rel="noopener nofollow ugc">Pages</a></h3>

<p>Pages are the most basic building block for content. They’re useful for standalone content (content which is not date based or is not a group of content such as staff members or recipes).</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>

**Comment by walkermoskop** [comment]
<p>Thank you! That did it.</p>
